src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java

```diff
@@ -0,0 +1,114 @@
+package org.folio.sample.integration.api.books;
+
+import io.restassured.common.mapper.TypeRef;
+import io.restassured.response.Response;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.UUID;
+
+import org.folio.sample.domain.dto.BookDTO;
+import org.folio.sample.domain.dto.BookForCreationDTO;
+import org.folio.sample.integration.AbstractBaseApiTest;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.http.HttpStatus;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.hasSize;
+
+
+
+class BookAvailableTest extends AbstractBaseApiTest {
+
+  @MethodSource
```

Comment:
Why is this here?

Reply:
Yea that was also copied over from the other test, but I should have removed all of that now.